src/main/java/frc/robot/RobotContainer.java

```diff
@@ -231,6 +232,10 @@ private void configureButtonBindings() {
 
     OI.getTrigger(OI.Operator.RTrigger).onTrue(climber.climb());
     OI.getTrigger(OI.Operator.LTrigger).onTrue(climber.retract());
+    OI.getTrigger(OI.Operator.RBumper).whileTrue(algeaRemover.goUp());
+    OI.getTrigger(OI.Operator.LBumper).whileTrue(algeaRemover.goDown());
+    OI.getTrigger(OI.Operator.A).onTrue(algeaRemover.stowAlgea());
+    OI.getTrigger(OI.Operator.B).onTrue(algeaRemover.removeAlgea());
```

Comment:
```suggestion
    OI.getButton(OI.Operator.RBumper).whileTrue(algeaRemover.goUp());
    OI.getButton(OI.Operator.LBumper).whileTrue(algeaRemover.goDown());
    OI.getButton(OI.Operator.A).onTrue(algeaRemover.stowAlgea());
    OI.getButton(OI.Operator.B).onTrue(algeaRemover.removeAlgea());
```
OI class strikes again!